src/Form/FormAnalyzer.js

```diff
@@ -9,6 +9,8 @@ const signupRegex = new RegExp(
 )
 const conservativeSignupRegex = new RegExp(/sign.?up|join|register|enroll|newsletter|subscri(be|ption)|settings|preferences|profile|update/i)
 const strictSignupRegex = new RegExp(/sign.?up|join|register|(create|new).+account|enroll|settings|preferences|profile|update/i)
+const resetPasswordLink = new RegExp(/(forgot(ten)?|reset|don't remember) (your )?password|password forgotten/i)
+const loginProvidersRegex = new RegExp(/ with /i)
```

Comment:
Changes here are unrelated to the mutating forms. Just improving accuracy and splitting these two regexes so we can use them independently.